Update migrating-to-github-container-registry-for-docker-images.md

danBamikiya opened the pull request:
This commit solves #5020. By preferring the use of the GITHUB_TOKEN instead of CR_PAT.

<!--
Thank you for contributing to this project! You must fill out the information below before we can review this pull request. By explaining why you're making a change (or linking to a pull request) and what changes you've made, we can triage your pull request to the best possible team for review.

See our [CONTRIBUTING.md](/main/CONTRIBUTING.md) for information how to contribute.

For changes to content in [site policy](https://github.com/github/docs/tree/main/content/github/site-policy), see the [CONTRIBUTING guide in the site-policy repo](https://github.com/github/site-policy/blob/main/CONTRIBUTING.md).

We cannot accept changes to our translated content right now. See the [contributing.md](/main/CONTRIBUTING.md#earth_asia-translations) for more information.

Thanks again!
-->

### Why:

<!-- 
- If there's an existing issue for your change, please link to it.
- If there's _not_ an existing issue, please open one first to make it more likely that this update will be accepted: https://github.com/github/docs/issues/new/choose. -->
**Closes #5020 **

### What's being changed:
![github_docs_contribution_1](https://user-images.githubusercontent.com/58262528/116720349-fb0c1a00-a9d3-11eb-8fa8-317ef962eba7.png)


<!-- Share artifacts of the changes, be they code snippets, GIFs or screenshots; whatever shares the most context. -->

### Check off the following:
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ x ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ x ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (eg. a "before and after video")

<!-- Description of the writer impact here -->

danBamikiya committed:
Update migrating-to-github-container-registry-for-docker-images.md

This commit solves #5020. By preferring the use of the GITHUB_TOKEN instead of CR_PAT.

janiceilene commented:
@danBamikiya Thanks so much for opening a PR! I'll get this triaged for review ⚡

jmarlena committed:
fix merge conflict

jmarlena committed:
Fix reusable

hubwriter commented on content/packages/working-with-a-github-packages-registry/migrating-to-the-container-registry-from-the-docker-registry.md:
```diff
@@ -95,9 +95,9 @@ If you have a {% data variables.product.prodname_actions %} workflow that uses a
 2. In your {% data variables.product.prodname_actions %} workflow file, update the package url from `https://docker.pkg.github.com` to `ghcr.io`.
 
-3. Add your new {% data variables.product.prodname_container_registry %} authentication personal access token (PAT) as a GitHub Actions secret. The {% data variables.product.prodname_container_registry %} does not support using `GITHUB_TOKEN` for your PAT so you must use a different custom variable, such as `CR_PAT`. For more information, see "[Creating and storing encrypted secrets](/actions/con ...
+3. Use the `GITHUB_TOKEN` for your authentication personal access token (PAT) as a GitHub Actions secret. {% data variables.product.prodname_container_registry %} now supports using `GITHUB_TOKEN` for your PAT. For more information, see "[Authenticating with the Github Token](/actions/configuring-and-managing-workflows/authenticating-with-the-github_token)."
```
I think we could shorten this step to just:
```suggestion
3. Use the `GITHUB_TOKEN` for your authentication personal access token (PAT). For more information, see "[Authentication in a workflow](/actions/reference/authentication-in-a-workflow)."
```

hubwriter commented on content/packages/working-with-a-github-packages-registry/migrating-to-the-container-registry-from-the-docker-registry.md:
```diff
@@ -95,9 +95,9 @@ If you have a {% data variables.product.prodname_actions %} workflow that uses a
 2. In your {% data variables.product.prodname_actions %} workflow file, update the package url from `https://docker.pkg.github.com` to `ghcr.io`.
 
-3. Add your new {% data variables.product.prodname_container_registry %} authentication personal access token (PAT) as a GitHub Actions secret. The {% data variables.product.prodname_container_registry %} does not support using `GITHUB_TOKEN` for your PAT so you must use a different custom variable, such as `CR_PAT`. For more information, see "[Creating and storing encrypted secrets](/actions/con ...
+3. Use the `GITHUB_TOKEN` for your authentication personal access token (PAT) as a GitHub Actions secret. {% data variables.product.prodname_container_registry %} now supports using `GITHUB_TOKEN` for your PAT. For more information, see "[Authenticating with the Github Token](/actions/configuring-and-managing-workflows/authenticating-with-the-github_token)."
```
I think we could shorten this step to just:
```suggestion
3. Use the `GITHUB_TOKEN` for your authentication personal access token (PAT). For more information, see "[Authentication in a workflow](/actions/reference/authentication-in-a-workflow)."
```

danBamikiya replied:
@hubwriter  Hi, I just did that!

hubwriter reviewed (commented):
@danBamikiya - Many thanks for raising this pull request. 
Your changes are great, thanks. I've suggested one small change. 
If you'd like to commit my suggestion we'll get this merged. 
Or feel free to make an alternative change if you prefer.

Thanks once again for helping to improve the GitHub documentation. 👍

danBamikiya committed:
Update container registry docs

This commit makes the migrating-to-github-container-registry-for-docker-images.md more descriptive about using the Github Token for workflow authentication.

Co-authored-by: hubwriter <hubwriter@github.com>

hubwriter commented:
@danBamikiya - Many thanks.
Merging this now. It will show up on the live site soon.
👏

hubwriter committed:
Merge branch 'main' into patch-1
